data/json/professions.json

```diff
@@ -2756,6 +2756,60 @@
       "female": { "entries": [ { "item": "bra" }, { "item": "panties" }, { "item": "gold_ear" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "santa",
+    "name": { "male": "Mall Santa", "female": "Mall Mrs. Claus" },
+    "description": "Everything was set for Christmas, you where just waiting for the call.  The children's happiness and joy where lost due to the Cataclysm, but you are decided to keep the Christmas Spirit alive, no matter what.  At least, all that time making toys and carrying them will be of use.",
```

Comment:
This does not depict someone working as a mall santa. Neither does the skill set.